src/site/content/en/blog/optimize-fid/index.md

```diff
@@ -0,0 +1,300 @@
+    supports this via [`target.esmodules`](https://babeljs.io/docs/en/babel-preset-env#targetsesmodules))	
+
+    ```html	
+    <script type="module" src="modern.js"></script>	
+    <script nomodule src="legacy.js" defer></script>	
+    ```	
+
+    Many newer ECMAScript features compiled with Babel are already supported in environments	
+    that support JavaScript modules. So by doing this, you simplify the process of making sure that	
+    only transpiled code is used for browsers that actually need it.
+
+{% Aside %}
+The [Serve modern code to modern browsers for faster page
+loads](/serve-modern-code-to-modern-browsers/) guide goes into more detail about this topic.
+{% endAside %}
+
+## Break up Long Tasks
+
+If you've already attempted to reduce the amount of JavaScript that loads on a single page, it can
+be useful to break down long-running code  into **smaller, asynchronous tasks**. 
+
+**Long Tasks** are JavaScript execution periods where users may
+find your UI unresponsive. Any piece of code that blocks the main thread for 50 ms or more can be
+characterized as a Long Task. Long Tasks are commonly a cause of poor FID scores and are a sign of
+potential JavaScript bloat (loading and executing more than a user may need right now).  
+Separating long tasks can reduce input delay on your site. 
+
+<figure class="w-figure">
+  <img src="./long-task.png" alt="Long Tasks in Chrome DevTools">
+  <figcaption class="w-figcaption">Chrome DevTools <a href="https://developers.google.com/web/updates/2020/03/devtools#long-tasks">visualizes Long Tasks</a> in the Performance Panel</figcaption>
+</figure>
+
+FID should improve noticeably as you adopt best practices like code-splitting and breaking up your
+Long Tasks. While TBT is not a field metric, it's useful for checking progress towards ultimately
+improving both Time To Interactive (TTI) and FID.
+
+{% Aside %}
+For more information, take a look at [Are long JavaScript tasks delaying your Time to
+Interactive?](/long-tasks-devtools/) guide goes into more detail about this topic.
+{% endAside %}
+
+## Optimize your page for  interaction readiness 
+
+There are a number of common causes for poor FID and TBT scores in web apps that rely heavily on
+JavaScript:
+
+### First-party script execution can delay interaction readiness
+
++   JS size bloat, heavy execution times and inefficient chunking can slow down how soon a
+    page can respond to user input and impact FID, TBT and TTI. Progressive loading of code and
+    features can help spread this work out and improve interaction readiness.
++   Server-side rendered apps may look like they're getting pixels painted on the screen
+    quickly, but beware user interactions being blocked by large script executions (e.g
+    re-hydration to wire up event listeners). This can take several hundred milliseconds, sometimes
+    even seconds, if route-based code-splitting is being used. Consider shifting more logic
```

Comment:
This makes it sounds like route-based code splitting is the problem.

Reply:
s/code-splitting/code splitting/

Reply:
```suggestion
    even seconds, if route-based code splitting is being used. Consider shifting more logic
```